src/Highlighter.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\SqlFormatter;
+
+interface Highlighter
+{
+    /**
+     * Highlights a token depending on its type.
+     *
+     * @param mixed[] $token An associative array containing type and value.
+     */
+    public function highlightToken(array $token) : string;
```

Comment:
Since `highlightToken` is used only in two places, why not refactor it already to be 
```
public function highlightToken(string $type, string $value) : string;
```

?

Reply:
I'm going to do that one change here and merge, since it's relatively small compared to the other one that I will do in a separate PR